### ISSUE TITLE
"Potential return stack corruption" exception raised when _game_menu_screen is None

### ISSUE THREAD

**Issue by jsfehler** [renpy/renpy#2613]
The following error occurs when reloading and when calling `ShowMenu()` outside the main menu screen, after calling a screen.

```Full traceback:
  File "renpy/common/00gamemenu.rpy", line 160, in script
    label _game_menu(*args, _game_menu_screen=_game_menu_screen, **kwargs):
  File "renpy/execution.py", line 435, in check_stacks
    raise e
Exception: Potential return stack corruption: dynamic=12 return=0
```

Issue encountered in version 7.4.1.1241, but not 7.4.0.

**Comment by renpytom** [comment]
This should be fixed by 3e614454e9b47bd36cf1054cf64ed8192f7c6a37 anmd a96a6c0ce30e3f3778136bf91c8ecb9abec3cf4f.

**Comment by jsfehler** [comment]
Fixed in 7.4.1.1259
